recipes/ipycache/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: ipycache
+  version: {{ version }}
+
+source:
+  fn: ipycache-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/i/ipycache/ipycache-{{ version }}.zip
+  md5: c78c6094ffab45b0e33c2f556705fa11
+
+build:
+  number: 0
```

Comment:
Please add `script: python setup.py install --single-version-externally-managed --record=record.txt`.

Then drop `build.sh` and `bld.bat`.